examples/BatteryMonitorLPP/BatteryMonitorLPP.ino

```diff
@@ -0,0 +1,172 @@
+// **********************************************************************************
```

Comment:
Although useful, we're not using these headers in examples.

Please add the concerning comments in code, if necessary.